### PR TITLE
fix(UI-1093): centralize the logo

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Align the logo to the center.

Before:

image

After:

image

## Linear Ticket

https://linear.app/autokitteh/issue/UI-1093/logo-not-centered

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [x] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/sidebar/sidebar.tsx
Comment (J1za) [inline src/components/organisms/sidebar/sidebar.tsx:1]: [screencast-localhost_8000-2024_12_18-18_27_58.webm](https://github.com/user-attachments/assets/4ea4a120-b6d6-4af0-986a-9c76fb21b577)

Comment (RonenMars) [reply src/components/organisms/sidebar/sidebar.tsx:1]: fixed

File: src/components/organisms/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -42,24 +42,26 @@ export const Sidebar = () => {
...
-										initial="hidden"
-										variants={animateVariant}
-									>
-										<IconLogoName className="h-3 w-20" />
-									</motion.span>
-								) : null}
-							</AnimatePresence>
-						</Link>
-
+						<div className="flex">
```
Comment (J1za) [inline src/components/organisms/sidebar/sidebar.tsx:45]:
```suggestion
						<div className="flex gap-1.5">
```

File: src/components/organisms/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -42,24 +42,26 @@ export const Sidebar = () => {
...
+							<Link className="ml-1 flex w-full justify-start gap-2.5" to="/">
```
Comment (J1za) [inline src/components/organisms/sidebar/sidebar.tsx:46]:
```suggestion
							<Link className="ml-1 flex justify-start gap-2.5" to="/">
```

**Review by J1za — APPROVED** [review]

**Pull request merged**